src/walletConnect/types.ts

```diff
@@ -1,3 +1,115 @@
+import { ChainId } from '@/chains/types';
+import { Address } from 'viem';
+import { SignClientTypes, Verify } from '@walletconnect/types';
+import { RequestSource } from '@/utils/requestNavigationHandlers';
+
+/**
+ * Display details loaded for a request.
+ */
+interface RequestDisplayDetails {
```

Comment:
moved some of the types from the `redux/requests` and `redux/walletconnect` over to this file instead